packages/client/tests/functional/issues/10229/tests.ts

```diff
@@ -33,8 +33,8 @@ testMatrix.setupTestSuite(
       `,
     },
     skipDataProxy: {
-      runtimes: ['edge'],
-      reason: 'InvalidDatasourceError will be shown on edge runtime, we should fix this in Prisma 6',
+      runtimes: ['node', 'edge'],
+      reason: 'InvalidDatasourceError is not compatible with asserted error // Change in Prisma 6',
```

Comment:
Previously removed, but now restored as the functional test suite is now testing with `--no-engine`. This kinda shows the bad discrepancy we have, but we can only get it fixed in Prisma 6 probably.